server/server.go

```diff
@@ -1,15 +1,39 @@
 package main
 
 import (
+	"net/http"
+
 	"github.com/labstack/echo/v4"
 )
 
+type MetaStruct struct {
+	Total int `json:"total"`
+}
+
+type TestJSON struct {
+	Meta MetaStruct `json:"meta"`
+	Data []string `json:"data"`
```

Comment:
IMO `data` should be an array of `Content` structs.  A `Content` struct may look something like this:

```go
type Content struct {
  Body string `json:"body"`
}
```

Making your return type for the content response be something more like...

```go
type ContentResponse struct {
  Meta ContentResponseMeta `json:"meta"`
  Data []Content `json:"data"`
}
```

I'd also rename `TestJSON` to be something that better describes its purpose.

Reply:
Would `ResponseJSON` be more appropriate? Or is it not really good/necessary to have the word 'JSON' in the struct name definition

Reply:
Hope my ContentResponse comment helped clarify this.